tools/webdriver/webdriver/client.py

```diff
@@ -752,6 +758,60 @@ def execute_async_script(self, script, args=None):
     def screenshot(self):
         return self.send_session_command("GET", "screenshot")
 
+class BidiSession(Session):
```

Comment:
nit: I think there would be something to be said for this being in a different file.

Reply:
Done.